charts/testkube-enterprise/local-values.yaml

```diff
@@ -223,6 +223,21 @@ testkube-cloud-api:
     apiAddress: "http://localhost:8090"
     # -- S3 bucket in which to store logs & artifacts
     outputsBucket: testkube-cloud-outputs
+    # -- External log server connection configuration
+    logServer:
```

Comment:
Logs service is not installed as part of the enterprise chart, it was causing issues with the certificate so it was taken out.

If you set this without the logs service will the logs be lost?

Reply:
I've conditionally enabled it - to be able to install on local enterprise. 

disabled by default in values.

Reply:
@emil2k also renamed tls secret name

Reply:
@exu Thank you, we are going to try to test your changes.